simpledotorg/simple-server
# Upgrade Bundler to 2.2.29

**Story card:** [ch3875](https://app.shortcut.com/simpledotorg/story/3875/upgrade-bundler-across-all-environments)

## Deployment
See the [story](https://app.shortcut.com/simpledotorg/story/3875/upgrade-bundler-across-all-environments) for the full task list for safely deploying this.  This PR should only be deployed *after* the Ansible side: https://github.com/simpledotorg/deployment/pull/371

## Because

Our version of Bundler is far behind latest, and we need to update so we can safely upgrade Ruby.  There are many changes between 1.17 and 2.2.29 - see the [bundler changelog](https://github.com/rubygems/rubygems/blob/3.2/bundler/CHANGELOG.md) or [upgrade docs](https://bundler.io/guides/bundler_2_upgrade.html) details.

## This addresses

Upgrading our Gemfile to Bundler 2.2.29.

## Test instructions

clone this branch, run `gem install bundler && bundle`, and verify you can start the server and run tests.
I've verified this works for me on my dev machine.
